recipes/yum_setup.rb

```diff
@@ -0,0 +1,15 @@
+#
+# Cookbook Name:: delugeserver
+# Recipe:: yum_setup
+#
+# Copyright (c) 2016 The Authors, All Rights Reserved.
+
+include_recipe 'yum'
```

Comment:
Yea no way to get around it on a "RPM" box. This and the installing the RPM repo are platform specific. When we go to implement other platforms we can use case statements to select which package manager to use based on platform.